instrumentation/trilogy/opentelemetry-instrumentation-trilogy.gemspec

```diff
@@ -27,14 +27,14 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'opentelemetry-api', '~> 1.0'
   spec.add_dependency 'opentelemetry-helpers-mysql'
+  spec.add_dependency 'opentelemetry-helpers-sql'
```

Comment:
Could you add the new helpers gem to the gemspec for mysql2 and pg as well?

Reply:
OMG DID I NOT

Reply:
I think adding the helpers gem to the Gemfile's `:test` group concealed the issue! The tests would load the gem through that path, so it didn't need to be in the main `gemspec`.

Reply:
♻️